--- gnome-control-center-3.24.2/panels/common/gsd-device-manager-udev.c	2017-05-10 10:10:21.000000000 +0200
+++ gsd-device-manager-udev.c	2017-06-07 22:57:50.817985244 +0200
@@ -24,7 +24,7 @@
 #include <string.h>
 #include <gudev/gudev.h>
 
-#include <gdk/gdkwayland.h>
+//#include <gdk/gdkwayland.h>
 #include "gsd-device-manager-udev.h"
 
 struct _GsdUdevDeviceManager
@@ -231,8 +231,8 @@
 	GHashTableIter iter;
 	GsdDevice *device;
 
-	node_path = gdk_wayland_device_get_node_path (gdk_device);
-	if (!node_path)
+	//node_path = gdk_wayland_device_get_node_path (gdk_device);
+	//if (!node_path)
 		return NULL;
 
 	g_hash_table_iter_init (&iter, GSD_UDEV_DEVICE_MANAGER (manager)->devices);

